OpenRA.Game/Settings.cs

```diff
@@ -207,6 +207,17 @@ public class KeySettings
 		public Hotkey MapScrollLeft = new Hotkey(Keycode.LEFT, Modifiers.None);
 		public Hotkey MapScrollRight = new Hotkey(Keycode.RIGHT, Modifiers.None);
 
+		public Hotkey ViewPortBookmarkSaveSlot1 = new Hotkey(Keycode.NUMBER_1, Modifiers.Alt | Modifiers.Shift);
+		public Hotkey ViewPortBookmarkSaveSlot2 = new Hotkey(Keycode.NUMBER_2, Modifiers.Alt | Modifiers.Shift);
+		public Hotkey ViewPortBookmarkSaveSlot3 = new Hotkey(Keycode.NUMBER_3, Modifiers.Alt | Modifiers.Shift);
+		public Hotkey ViewPortBookmarkSaveSlot4 = new Hotkey(Keycode.NUMBER_4, Modifiers.Alt | Modifiers.Shift);
+
+		public Hotkey ViewPortBookmarkUseSlot1 = new Hotkey(Keycode.NUMBER_1, Modifiers.Alt);
```

Comment:
This will conflict with the control groups: alt + number is used to center the viewport on the numbered group.